_config.yml

```
@@ -0,0 +1,26 @@
+title: USBPcap
+email: your-email@example.com
```

^

---

Feel free to use
```suggestion
suggestions
```
and merge it yourself :-)